Empty                         req_data
---
Empty                         res_data
bool                          ark_service_timeout

